hdr/sqlite_modern_cpp.h

```diff
@@ -405,41 +405,61 @@ namespace sqlite {
 		);
 	}
 	
+	enum class Flags {
+		OPEN_READONLY = SQLITE_OPEN_READONLY,
+		OPEN_READWRITE = SQLITE_OPEN_READWRITE,
+		OPEN_CREATE = SQLITE_OPEN_CREATE,
+		OPEN_NOMUTEX = SQLITE_OPEN_NOMUTEX,
+		OPEN_FULLMUTEX = SQLITE_OPEN_FULLMUTEX,
+		OPEN_SHAREDCACHE = SQLITE_OPEN_SHAREDCACHE,
+		OPEN_PRIVATECACH = SQLITE_OPEN_PRIVATECACHE,
+		OPEN_URI = SQLITE_OPEN_URI
+	};
```

Comment:
@aminroosta What do you think about omitting OPEN_? It looks a bit redundant. If you prefer to keep it we might move it to the class name: `OpenFlags::READONLY` looks better than `Flags::OPEN_READONLY`